rocketpy/environment/environment_analysis.py

```diff
@@ -8,25 +8,24 @@
 import numpy as np
 import pytz
 
-from rocketpy.Environment import Environment
-from rocketpy.Function import Function
-from rocketpy.units import convert_units
-
-from .plots.environment_analysis_plots import _EnvironmentAnalysisPlots
-from .prints.environment_analysis_prints import _EnvironmentAnalysisPrints
-from .tools import (
+from ..mathutils.function import Function
+from ..plots.environment_analysis_plots import _EnvironmentAnalysisPlots
+from ..prints.environment_analysis_prints import _EnvironmentAnalysisPrints
```

Comment:
I am just commenting on these lines, however this happens in most import statements regarding the new structure.

Here there was a change from absolute imports to relative imports. Of course, there are use cases for both of them and I would like to point out the main advantages/disadvantages of each option:

1. See the discussion ["Absolute vs relative imports"](https://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module)
2. Despite being very old (but still active), the import section of [PEP8](https://peps.python.org/pep-0008/#imports) (interesting read) used to strongly recommend absolute imports for packages. This comes from earlier Python versions in which relative imports were more troublesome than today. Nowadays I believe there is not a clear recommendation, and both ways are acceptable; 
3. Relative imports have the advantage of being shorter and requiring less refactoring if the submodules are rearranged.

Personally, I believe we should keep this relative import implementation since it does not appear to be frowned upon anymore and having some interesting benefits. I just wanted to make sure that the existence of an alternative is acknowledged.

Reply:
Good comment, 

This is the first time I see this kind of discussion.
I am aware of this now. In the future we can modify the imports if we find it confusing for any reasons.

I agree moving forward with the current relative import.